cube_browser/meta.yaml

```diff
@@ -7,16 +7,17 @@ source:
     url: http://github.com/SciTools/cube_browser/archive/master.tar.gz
 
 build:
-    number: 0
+    number: 1
 
 requirements:
     build:
         - python
         - setuptools
     run:
+        - ipywidgets 5.1*
         - iris 1.10*
-        - ipywidgets 4.1*
         - matplotlib
+        - sphinx
```

Comment:
Probably not the right thing here. sphinx isn't a *runtime* dependency. If a user wants sphinx in their environment:

```
conda install cube_browser sphinx
````